reflex-dom-core/src/Reflex/Dom/Main.hs

```diff
@@ -15,6 +15,8 @@ module Reflex.Dom.Main where
 
 import Prelude hiding (concat, mapM, mapM_, sequence, sequence_)
 
+import Reflex.Class (holdDyn)
+import Reflex.Dom.Builder.Class (MountState(Mounted, Mounting))
```

Comment:
I run stylish-haskell as `stylish-haskell -i **/*.hs`, after I've committed everything to git, and then I check the git diff.  It's not ideal, but it works pretty well and makes it easy to revert.